src/Ztm.Data.Entity/Contexts/MainDatabase.cs

```diff
@@ -138,6 +142,30 @@ protected virtual void ConfigureWebApiCallbackHistory(EntityTypeBuilder<WebApiCa
                    .OnDelete(DeleteBehavior.Cascade);
         }
 
+        protected virtual void ConfigureReceivingAddress(EntityTypeBuilder<ReceivingAddress> builder)
+        {
+            builder.Property(e => e.Id).IsRequired().ValueGeneratedOnAdd();
+            builder.Property(e => e.Address).IsRequired();
+            builder.Property(e => e.IsLocked).HasDefaultValue(false);
+
+            builder.HasKey(e => e.Id);
+            builder.HasIndex(e => e.Address).IsUnique();
+        }
+
+        protected virtual void ConfigureReceivingAddressReservation(EntityTypeBuilder<ReceivingAddressReservation> builder)
+        {
+            builder.Property(e => e.Id).IsRequired().ValueGeneratedOnAdd();
+            builder.Property(e => e.ReceivingAddressId).IsRequired();
+            builder.Property(e => e.LockedAt).IsRequired();
```

Comment:
We need to add `ReleasedAt` too even we have nothing to configure so it will be more easy to know how much columns it have.